pkg/entrypoint/entrypointer_test.go

```diff
@@ -204,6 +213,39 @@ func TestEntrypointer(t *testing.T) {
 	}
 }
 
+func assertExist(t *testing.T, entries []v1alpha1.PipelineResourceResult, key ...string) {
+
+	index := make(map[string]struct{}, len(entries))
+
+	for _, result := range entries {
+		index[result.Key] = struct{}{}
+	}
+
+	for _, item := range key {
+		if _, ok := index[item]; !ok {
+			t.Error("Didn't find the entry: " + item)
+		}
+	}
+}
+
+func assertEqual(t *testing.T, entries []v1alpha1.PipelineResourceResult, parameterValues ...string) {
```

Comment:
This could possibly cmp.Diff instead?

Reply:
en.. It is not a logic same as cmp.Diff,  the purpose is to judge that  element vlaue in `[]v1alpha1.PipelineResourceResult` is as expected.
According previous comment, I refect it inline, we dont need this assert-style helpers in latest commit.